e2e/v8-coverage/no-sourcemap/cssTransform.js

```diff
@@ -1,11 +0,0 @@
-/**
```

Comment:
please keep this test as is

(I'm fine with moving it, but don't delete/change it)